ledger-core/v2/application/functest/api_test_utils.go

```diff
@@ -21,7 +21,7 @@ import (
 
 var (
 	httpClient *http.Client
-	nodesPorts = [5]string{"32301", "32302", "32303", "32304", "32305"}
+	nodesPorts = [2]string{"32302", "32304"}
```

Comment:
Should be part of that PR?

Reply:
@bigbes Please, check in https://insolar.atlassian.net/browse/PLAT-261

Reply:
"that"? Which PR do you mean by "that"?

Task PLAT-261 mentions this part. Have you read description of the task?